orion/sites/admin/src/announce/announce.js

```diff
@@ -2,12 +2,70 @@
 require('./announce.styl');
 import React from 'react';
 import ReactDOM from 'react-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
```

Comment:
You can remove API and Modal, they're not being used yet. Only include what you need.